index.bs

```diff
@@ -384,6 +384,56 @@ scheme, is encoded in the <code>URL</code> field.
   };
 </pre>
 
+<div class="example">
+In this example we build a page supporting a device. To start with when the page
+is loaded it should first check if the device is already plugged in and it has
+permission to access it by calling {{USB/getDevices()}},
```

Comment:
To avoid "it has permission to access it", maybe "if it already has permission to access a plugged-in device by calling ..."